src/include/iscsi-client/helps.rb

```diff
@@ -26,14 +26,26 @@
 # Summary:	Help texts of all the dialogs
 # Authors:	Michal Zugec <mzugec@suse.cz>
 #
-# $Id$
 module Yast
   module IscsiClientHelpsInclude
     def initialize_iscsi_client_helps(include_target)
       textdomain "iscsi-client"
 
+      Yast.import "Arch"
+
       use_bidirectional_auth = _("If authentication is needed for secure access, please use <b>Authentication by Initiator</b> and <b>Authentication by Targets</b> " \
       "together. Please do not only use one of them for security reasons.\n")
+
+      startup_help = lambda do
```

Comment:
NP: This is technically correct but it took me a while to find out why this lambda and a temporary variable are needed...
(It's because of that quite hidden `if !Arch.s390` condition at the end of line.)

In this case it would be IMHO a good idea to add here a short comment like `the help text is architecture dependent` or something like that as this is quite unusual.

Reply:
@lslezak No, it is independent of the Arch condition. AFAIK the lambda is a way to make a nested `def`, which @aschnell prefers to a top-level def.